code/datums/status_effects/buffs.dm

```diff
@@ -421,11 +421,9 @@
 
 
 //Used by changelings to rapidly heal
-//Heals 10 brute and oxygen damage every second, and 5 fire
 //Being on fire will suppress this healing
 /datum/status_effect/fleshmend
 	id = "fleshmend"
-	duration = 100
```

Comment:
You are aware this makes the effect never end until turned off by the user right?

Reply:
yes, that was intentional